aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -668,7 +668,7 @@ Generates the following HTML:
 
 If you find yourself using the "not specified" option in multiple pages, you can create a template to eliminate repeating the HTML:
 
-[!code-cshtml[](../../mvc/views/working-with-forms/sample/final/Views/Home/IndexEmptyTemplate.cshtml?highlight=5)]
+[!code-cshtml[](../../mvc/views/working-with-forms/sample/final/Views/Home/IndexEmptyTemplate.cshtml?highlight=4)]
```

Comment:
Fixes the highlighting:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/a043b0d8-e12a-49da-882e-a50e815b84bb)